incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -460,13 +471,14 @@ private void onBegin(
             if (allowedBinding.test(begin.routedId()))
             {
                 final MutableDirectBuffer buffer = new UnsafeBuffer(ByteBuffer.allocate(begin.sizeof()));
-                final BeginFW f = new BeginFW.Builder().wrap(buffer, 0, begin.sizeof()).set(begin).build();
-                final ExtensionFW extension = f.extension().get(extensionRO::tryWrap);
+                begin2RO = new BeginFW.Builder().wrap(buffer, 0, begin.sizeof()).set(begin).build();
```

Comment:
These builders should be hoisted to private member fields, e.g.  beginRW` flyweight builder.

The mutable direct buffer should be pre-created once and reused too.

Reply:
OK